translate/noxfile.py

```diff
@@ -118,11 +118,18 @@ def system(session):
     session.install("-e", "../test_utils/")
     session.install("-e", ".")
 
+    # Additional setup for VPCSC system tests
+    env = {
+        "PROJECT_ID": os.environ.get("PROJECT_ID"),
+        "GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_PERIMETER_PROJECT": "570941833855",
+        "GOOGLE_CLOUD_TESTS_IN_VPCSC": "true",
```

Comment:
I am not sure this should be hardcoded. Developers on local test project don't necessarily have a VPCSC configured.

Reply:
I removed those hard-coded env params